src/strategies/soundxyz-metadata/transformer.mjs

```diff
@@ -0,0 +1,44 @@
+// @format
+import { decodeCallOutput } from "eth-fun";
+
+export function transform(line) {
+  let editionMetadataObject = {};
+
+  try {
```

Comment:
A `try {} catch` block must only contain a single statement (e.g. `decodeCallOutput`).

Reply:
Why hasn't this been resolved?

Reply:
I have resolved it in https://github.com/neume-network/strategies/pull/54/commits/8681a29311bde62de2b0be72d1c9f7f3e1754996. Let me know if this isn't what you wanted.